### ISSUE TITLE
D-FINE-M predictions for 16-bit gray show lots of low confidence

### ISSUE THREAD

**Issue by piotrgrubicki** [open-edge-platform/training_extensions#6725]
rc1
1. Create a project with 16-bit gray (one channel) png images.
2. Annotate three of them.
3. Start the D-FINE-M training.
4. Once finished, double click on an unannotated image.
5. It opens in the prediction mode. Check the result.

Lots of (as shown below) predictions with extremely low confidence. I'd expect them to be cut off before presenting.

`<confidence_threshold value="0.05" />` in the exported model probably explains it.

<img width="50%" height="50%" alt="Image" src="https://github.com/user-attachments/assets/643cecf5-6e7a-43a7-9fae-d7b5142ca2d9" />

**Comment by leoll2** [comment]
Your training dataset (3 images) is too small to expect good quality predictions. Please try with a larger dataset and reopen the ticket if the issue persists.

**Comment by piotrgrubicki** [comment]
@leoll2, this ticket is not about low quality. It is about the confidence set to 5% in the exported model.

**Comment by leoll2** [comment]
That's also a side effect of the tiny dataset size: the "optimal" confidence threshold cannot be reliably determined when the validation set is so small. The model couldn't learn much from 1 image, so it will return bad predictions regardless of the threshold value.

Moreover, the confidence score / threshold doesn't really have a physical meaning, it's just a parameter and it's heavily affected by the dataset and model architecture. In practice, you can find models that output qualitatively good predictions with low confidence, or viceversa.
